Baraka-Mugisha/MyDiary
# bug(APIfeedback): Fixing back end bugs and refactoring

#### What does this PR do?
`Responds to the feedback about back end codes`

#### Description of Task to be completed?
`remove validations from the controllers`
`remove return statements from controllers`
`fix entries with the same id`
`refactoring`

#### How should this be manually tested?
`clone the repo`
`do npm install`
`check the codes`
`install POSTMAN and tests all the routes`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#169497693

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
